ui/lib/core/stories/list-view.stories.js

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable import/extensions */
+import hbs from 'htmlbars-inline-precompile';
+import { storiesOf } from '@storybook/ember';
+import { withKnobs, select } from '@storybook/addon-knobs';
+import notes from './list-view.md';
+import ArrayProxy from '@ember/array/proxy';
+
+let filtered = ArrayProxy.create({ content: [] });
+filtered.set('meta', {
+  lastPage: 1,
+  currentPage: 1,
+  total: 100,
+});
+
+let paginated = ArrayProxy.create({
+  content: [{ id: 'middle' }, { id: 'of' }, { id: 'the' }, { id: 'list' }],
+});
+paginated.set('meta', {
+  lastPage: 10,
+  currentPage: 4,
+  total: 100,
+});
+
+let items = {
```

Comment:
i noticed in the JSDoc comments that `items` was supposed to be an array, though this is an object. can the component handle either type? or will `items` always be an object?

Reply:
These are the items options for the storybook select knob, I'll rename it so that's more apparent. The component usually takes an array though, but RecordArray can have `meta` items associated to on it.